StaticAnalyzer/views/android/cert_analysis.py

```diff
@@ -54,43 +52,20 @@ def cert_info(app_dir, app_file, tools_dir):
     """Return certificate information."""
     try:
         logger.info('Reading Code Signing Certificate')
-        cert = os.path.join(app_dir, 'META-INF/')
-        cp_path = tools_dir + 'CertPrint.jar'
-        files = [f for f in os.listdir(
-            cert) if os.path.isfile(os.path.join(cert, f))]
-        certfile = None
         dat = ''
         manidat = ''
-        manifestfile = None
-        if 'CERT.RSA' in files:
-            certfile = os.path.join(cert, 'CERT.RSA')
-        else:
-            for file_name in files:
-                if file_name.lower().endswith('.rsa'):
-                    certfile = os.path.join(cert, file_name)
-                elif file_name.lower().endswith('.dsa'):
-                    certfile = os.path.join(cert, file_name)
-        if certfile:
-            args = [settings.JAVA_BINARY, '-jar', cp_path, certfile]
+        issued = ''
+
+        dat = androguard_certinfo(app_dir, app_file)
+
+        if re.findall(r'Apk is signed', dat):
```

Comment:
This is not a good idea. What if the result has that string somewhere else. The value from the `is_signed()`

Reply:
will merge function in PR #1036